### ISSUE TITLE
CDN build error for @polkadot/rpc-core@6.12.2-17

### ISSUE THREAD

**Issue by danforbes** [jspm/project#156]
_Reporting CDN Build Error._

**Comment by guybedford** [comment]
There may well be a few more of these - if there are more than a couple I'll create a new issue to work through the build error queue.

**Comment by guybedford** [comment]
This one should be working now though.

**Comment by danforbes** [comment]
I'm trying to use `@polkadot/api` and it still doesn't seem to be working 😣 Thanks for your attention so far 🙏🏻 Here is the web UI you provide (https://generator.jspm.io/#U2VhYGBkDM0rySzJSU1hcCjIz8lOTMkv0U8syHQw0zM00jPSNTQHAAirKSEmAA):

![image](https://user-images.githubusercontent.com/2193247/146653701-891f1450-be8a-4363-ace8-286a54c95852.png)

Via Deno, it just hangs here:

![image](https://user-images.githubusercontent.com/2193247/146653711-d299d2f5-6e8c-43b5-9367-9d103e7d8faa.png)

**Comment by guybedford** [comment]
No problem, finally got this working via https://generator.jspm.io/#U2VhYGBkDM0rySzJSU1hcCjIz8lOTMkv0U8syHQw0zM00jPSNTQHAAirKSEmAA but it's giving an issue [in the sandbox](https://jspm.org/sandbox#H4sIAAAAAAAAA81aWXPbNhB+z69A1JekNQ9Jcaw4kkfO0Zxukypup08diFyZsEiCA0BXOv3vXYKkK9EUBUpqJg+JSGL3+/YiiIXRf+hzT60SIIGKwosH/eIHqH/xgJB+BIoSL6BCghq0Zmpi9Vp6QDEVwsV1rH/9vpPd36nENIJBa85gkXChWsTjsYIYIRbMV8HAhznzwNI3J4TFTDEaWtKjIQzaSNB3Mgv6Y+6vNOpDy8IfQt6PPl2RNxCDoIoL8i5K8ckVTfToa58pcv3bx3P0RyXnjhNyxAy4VOc9t+c6ksb+mC/t1M0frju/B3++eTF9deWK1ejr+9F1O/Be3r772gt//XI1nbvXPbl6+3nhfr1y3dtPo1++fH57ecnEh9Hr6PIS+SxLmyY9wRJF0jAOWkwbFNFEh+lvbVX+ULbO8wf4aJjwcEp9rhyaMBxopQZLtPiG2rcyiWzGnTiJzjfkhk/tdsfuWO0zh8U+LFGypQH/OcmIpMcT2OCpgF0bTg0Z0zGEjphhKiPAuIcJCOmAjBwvpFJ+EmxOFfzMIPQ/ci7hBZVQZ/AG3vDMbj+1Tw1hU39ODjDtA6z+B8sQtWzYelIsH1CjNiQV4lWp3MHgJNSb0ht4F0/4/mxrIFs5p7ASLL4xIsllhz3btbs7fIlBLbiYSiPgQhiR23bHEBnFFIgJ9aAxiatJCu3dTE2zUaIyyYNIPMvjwqy2CmHjyioUGntSwWTqTSL4nGFNGvMUCo28KpT28qyC0cS7dPo3qzktaeyPlsZw++CZo2fy+3A0Dlk1n3HELE8AzrZNqLRCU9+00r7OlRnNvZvGfBE3oNLyDX3TOnu6VuYz9mw/voZMM8VCI/BUUM+rT3ZELBXEfK4SZWb1mvzdJ8IQv3GEylxdoxgtqIyaeLQmP3xin9rtHR6tyVt4eWs2x93TyrkgStT21dQ9rcYx3MbbMJJWet2YML3O+XyqqDHRQV6ukZo4ubQmoLzAiCaXzRdgY8EXEkQN8k3Ix9TshS2EjRZ3S0tg38ajOQ1nht/YTZX81d3tgYKlwg8ZN12hbGjkE5AZC8RNWXINY5aFaaQWsjbD49iufen1OFYgTuyuE7KxkymsIwieZqMOI5MYttEOt7IaxrPJpD5c6Y2jl6VoiO1aY8BXsN11YowamiUL95wMqoS/ewY1xs+gSvjgd7yZqG0TC5EhTv0YheK2DBSGLFGsbjFYiODX9hRLJc2JD3O7eFwGnAN2PRFT2Pp061DX5TDfrl29PAmoDOorJpfQ2T7T5hU66zAR9SxfjOu60DsZhHJxAtRQxbMSHosDbIVVnV2FiE5wupLIbv+qfi9YYgm4gWUdYC6C0epuqeuIeYJb4/o9lf+EsDaebUWKWURDJjHF+O1Dg1Ut5D3pPIg7sfMPT7pcqYvmVp21ZOn7EpdY1hZPOjw8s7H+HB+x032b00qblUSGGiA9jqlOZxx9bYN8WsZYAKiYenUftTsZ7RfOB3hpTahUd7tzd/tz1ftxdfsppb26A1bGu/ZmDl0Zd0tbko2c3tJnbfP+CD1x59jmbs/V92FtudPbYe1hXflBxXC/5zreW3BQIO+X/DbDjrId2f1225FmrebxJ53jrOy76yvX/dK6o9CO0RmY9h+HdQadI8Vie20fIxbdbxKLA+uiKoXbg7Jf83vgjFQV2eObeNBsXtUiH9/EJ4ebuPkWfZeJ3izu7zLRm7sU3yTRD4r/0399J/tzfPrX9/zcAHk9Ilfcn4VARgGL5HlxaiCiiSRIs5qwMCQTLkikxSTJ5w1JFkwFfKYIW9OQs0TfPKKoBUsPEkVeBoJHQHrPfnpcOhZA5Sr2iBTeYGtjLa2M15Kpedji4RqqV3Q3G2M2NlWp6wT7QSm5YDcsHrRozONVxGeydVF2f+NsQoakDyaQwqMf0UBymTAyQQfKxxKeZyiEOA75A1thwACwOI8SSRNNsEWGE4KhQxhKJCRU4DqQYERBBw8fttD5FqFKCTae4RiPiQqgQMkstDWNx2PJQ7BDfvMIbXr8fCOhfSc7DdJ3slMq/wJMY3stvSIAAA==) that there's an export missing.

As far as I can tell though, that issue is a valid named export bug - @poladot/types-create doesn't have a 'mapXcmTypes' export at all, so perhaps this is something that is just passing Webpack / Rollup builds since they don't do strict checking or something like that?

**Comment by danforbes** [comment]
cc @jacogr

**Comment by jacogr** [comment]
I would certainly not suggest building any of the current bets - there is work ongoing for a new major release, so all these are expected. Stick to the stable versions - in this case I don't even pull in the betas into the other `@polkadot` projects at this state, there are too many breakages.

TL;DR All expected, these are marked beta for a reason :)
